lib/ljglibs/cdefs/fontconfig.moon

```diff
@@ -0,0 +1,9 @@
+-- Copyright 2015 The Howl Developers
```

Comment:
Ah that's where it was first before I decided it belongs with the g family since it's a dependency. :) 

I'll move it back.